packages/platform-test-suite/test/functional/core/getTransaction.spec.js

```diff
@@ -48,7 +45,7 @@ describe('Core', () => {
 
         expect.fail('should throw NotFound');
       } catch (e) {
-        expect(e).to.be.an.instanceOf(NotFoundError);
+        expect(e.constructor.name === 'NotFoundError');
```

Comment:
Maybe DAPI Client should export errors too? We could have collisions if we check by name only.

Reply:
Done